recipes/busybox/busybox_1.25.1.oe

```diff
@@ -3,5 +3,4 @@ require busybox.inc
 SRC_URI += "file://dhclient-options.patch"
 SRC_URI += "file://mdev-create-devices-from-sys-dev.patch"
 
-RECIPE_FLAGS += "s6rc"
```

Comment:
should this really be removed with the rest of the s6 related logic still there?